build_tools/build_linux_package.sh

```diff
@@ -0,0 +1,178 @@
+# Copyright 2025 The IREE Authors
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# Modified version of https://github.com/nod-ai/shark-ai/blob/main/shortfin/build_tools/build_linux_package.sh
```

Comment:
If this is forked from github.com/nod-ai/shark-ai we should retain the original file header.

Reply:
As Scott mentioned

> A build script can look like https://github.com/nod-ai/shark-ai/blob/main/shortfin/build_tools/build_linux_package.sh and a workflow could look like https://github.com/nod-ai/shark-ai/blob/main/.github/workflows/build_packages.yml.

but you can also look into the build script used in IREE: https://github.com/iree-org/iree/blob/main/build_tools/python_deploy/build_linux_packages.sh